src/_base/harness/config/functions.yml

```diff
@@ -80,3 +80,11 @@ function('php_fpm_exporter_scrape_url', [hostname, pools]): |
       )
     );
   = $text;
+
+function('version_compare', [version1, version2, operator]): |
+  #!php
+  $count1 = substr_count($version1, '.');
+  $count2 = substr_count($version2, '.');
+  $version1 .= $count1 == 1 ? '.0' : '';
+  $version2 .= $count2 == 1 ? '.0' : '';
```

Comment:
seems what you need to balance is the same number of components in each, not specifically more than one component